src/App.tsx

```diff
@@ -311,35 +341,35 @@ export default function App() {
                   />
                   <Redirect
                     from="/covid-infection-rate"
-                    to="/covid-risk-levels-metrics#icu-capacity-used"
+                    to="/covid-community-level-metrics#icu-capacity-used"
                   />
                   <Redirect
                     from="/what-is-covid-incidence"
-                    to="/covid-risk-levels-metrics#daily-new-cases"
+                    to="/covid-community-level-metrics#daily-new-cases"
                   />
                   <Redirect
                     from="/new-daily-covid-cases"
-                    to="/covid-risk-levels-metrics#daily-new-cases"
+                    to="/covid-community-level-metrics#daily-new-cases"
                   />
                   <Redirect
                     from="/old-hospitalization-projections"
-                    to="/covid-risk-levels-metrics#icu-capacity-used"
+                    to="/covid-community-level-metrics#icu-capacity-used"
                   />
                   <Redirect
                     from="/daily-new-cases-explained"
-                    to="/covid-risk-levels-metrics#daily-new-cases"
+                    to="/covid-community-level-metrics#daily-new-cases"
                   />
```

Comment:
Same comment as above. I think `#daily-new-cases` hash is dead.

Reply:
TBH we could probably remove all 8 of these redirects.  Seems unlikely anybody is linking to them anymore...  Else if we keep them, I'd update "daily-new-cases" to "weekly-new-cases."

Reply:
Let's get rid of them 👍🏼